HKMP/Game/Client/Entity/WanderingHusk.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections;
+using System.Collections.Generic;
+using Hkmp.Fsm;
+using Hkmp.Networking.Client;
+using Hkmp.Util;
+using UnityEngine;
+
+namespace Hkmp.Game.Client.Entity {
+    public class WanderingHusk : HealthManagedEntity {
+        private readonly PlayMakerFSM _fsm;
+
+        private Animation _lastAnimation;
+
+        private Walker _walker;
```

Comment:
This field and the `_audioSource` field can be made `readonly`.

Reply:
I agree. This has been resolved.